raiden/smart_contracts/NettingChannelLibrary.sol

```diff
@@ -157,31 +172,36 @@ library NettingChannelLibrary {
         Participant storage node1 = participants[0];
         Participant storage node2 = participants[1];
 
-        if (callerAddress != node1.nodeAddress && callerAddress != node2.nodeAddress) {
+        if (caller_address != node1.node_address && caller_address != node2.node_address) {
             throw;
         }
 
         (transfer_raw, transfer_address) = getTransferRawAddress(signed_transfer);
 
-        if (node1.nodeAddress == transfer_address) {
+        if (node1.node_address == transfer_address) {
             Participant storage sender = node1;
-        } else if (node2.nodeAddress == transfer_address) {
+        } else if (node2.node_address == transfer_address) {
             sender = node2;
         } else {
             throw;
         }
 
-        decode_and_assign(sender, transfer_raw);
+        decodeAndAssign(sender, transfer_raw);
 
-        self.closingAddress = callerAddress;
+        self.closing_address = caller_address;
         self.closed = block.number;
     }
 
     /// @notice close(bytes, bytes) to close a channel between to parties
     /// @dev Close the channel between two parties
-    /// @param firstEncoded (bytes) the last sent transfer of the msg.sender
-    /// @param secondEncoded (bytes) the last sent transfer of the msg.sender
-    function close(Data storage self, address callerAddress, bytes firstEncoded, bytes secondEncoded) {
+    /// @param first_encoded (bytes) the last sent transfer of the msg.sender
+    /// @param second_encoded (bytes) the last sent transfer of the msg.sender
+    function close(
+        Data storage self,
+        address caller_address,
+        bytes first_encoded,
+        bytes second_encoded
+    ) {
```

Comment:
@LefterisJP I think these changes got lost in my rebase chaos earlier, because I remember doing them. Nonetheless, I've pushed them again now and learned an good git lesson :)